openslides_backend/action/actions/user/update.py

```diff
@@ -49,21 +45,3 @@ class UserUpdate(
             "is_demo_user",
         ],
     )
-
```

Comment:
I think this is not what we discussed: Of course, we can model it this way, but we should really write it down somewhere how it's supposed to be. I understood it that all users given in `manager_ids` are just also added as members to the committee. Then, this check can stay and should work for the new code (except that you need to use `fetch_model` instead of `get`. @tsiegleauq @FinnStutzenstein Is this correct or do I remember it wrong?

Reply:
Re added the user check. And add the manager to the users of a committee.